packages/m365_defender/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.5.1"
+  changes:
+    - description: fix process.command_line overwritten by process.parent.command_line
+    type: bug
+    link: https://github.com/elastic/integrations/pull/8872
```

The CI seems to be failing here due to formatting 
`
Error: checking package failed: formatting the integration failed (path: /opt/buildkite-agent/builds/bk-agent-prod-gcp-1705066585826701134/elastic/integrations/packages/m365_defender, failFast: false): walking through the integration files failed: formatting file failed (path: /opt/buildkite-agent/builds/bk-agent-prod-gcp-1705066585826701134/elastic/integrations/packages/m365_defender/changelog.yml): formatting file content failed: unmarshalling YAML file failed: yaml: line 3: did not find expected '-' indicator
`

```suggestion
- version: "2.5.1"
  changes:
    - description: fix process.command_line overwritten by process.parent.command_line
      type: bugfix
      link: https://github.com/elastic/integrations/pull/8872
```
Here's the suggested change to format it correctly. You can run the command ` elastic-package build && elastic-package format && elastic-package lint && elastic-package check` and ensure this passes after the change.